app/src/main/res/values/themes.xml

```diff
@@ -1,62 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <!-- Base application theme. -->
-    <style name="Theme.MP" parent="Theme.Backbone">
```

Comment:
I think these are needed for Surveys, why did you remove them?

Reply:
These are duplicated in `researchstack-mpower`, so I moved them there